src/main/java/org/codehaus/plexus/components/io/filemappers/SuffixFileMapper.java

```diff
@@ -0,0 +1,74 @@
+package org.codehaus.plexus.components.io.filemappers;
+
+import java.util.regex.Matcher;
+
+/*
+ * Copyright 2007 The Codehaus Foundation.
```

Comment:
I've just noticed that this is the old license header. Judging by the other new files in the codehaus-plexus project you can just delete this line and keep the rest (the Apache license).